test/e2e/snapshot/snapshot.go

```diff
@@ -117,15 +199,36 @@ var _ = ginkgo.Describe("Snapshot VCluster", func() {
 		err = cmd.Run()
 		framework.ExpectNoError(err)
 
-		// now delete the service
-		err = f.VClusterClient.CoreV1().Services("default").Delete(f.Context, "snapshot-restore", metav1.DeleteOptions{})
+		// now create a configmap that should be deleted by restore
+		_, err = f.VClusterClient.CoreV1().ConfigMaps(defaultNamespace).Create(f.Context, configMapToDelete, metav1.CreateOptions{})
+		framework.ExpectNoError(err)
+
+		// now create a secret that should be deleted by restore
+		_, err = f.VClusterClient.CoreV1().Secrets(defaultNamespace).Create(f.Context, secretToDelete, metav1.CreateOptions{})
+		framework.ExpectNoError(err)
+
+		// delete deployment that should be restored
+		err = f.VClusterClient.AppsV1().Deployments(defaultNamespace).Delete(f.Context, deploymentToRestore.Name, metav1.DeleteOptions{})
 		framework.ExpectNoError(err)
 
-		// now create a service that should not be there when we restore
-		_, err = f.VClusterClient.CoreV1().Services("default").Create(f.Context, &corev1.Service{
+	})
+
+	ginkgo.AfterAll(func() {
+		// delete the snapshot pvc
+		err := f.HostClient.CoreV1().PersistentVolumeClaims(pvc.Namespace).Delete(f.Context, pvc.Name, metav1.DeleteOptions{})
+		framework.ExpectNoError(err)
+	})
+
+	ginkgo.It("Restore should overwrite data", func() {
```

Comment:
Can we have the testcase say "Restore should restore resources in the vCluster" instead?